docker/cami-store/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM mysql:5.6
+
+# Add the store schema to the container to be run when the first container is
+# run. This requires that the build context is the project root folder. Run
+# build like:
+# `docker build -f docker/cami-store/Dockerfile .`
+ADD store/cami.sql /docker-entrypoint-initdb.d/
+
+# Hardcode passwors until we learn a better way to do this.
```

Comment:
I read something about such a service: `confd`. It seems to allow creating template config files on the container which it populates with ENV vars taken from the container run command.